spec/acceptance/new_class_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper_acceptance'
+
+describe 'Creating a new class' do
+  before(:all) do
+    shell_ex("#{path_to_pdk} new module foo --skip-interview")
+  end
+
+  after(:all) do
+    shell_ex('rm -rf foo')
+  end
+
+  context 'when creating the main class' do
+    describe command("cd foo && BUNDLE_GEMFILE=../Gemfile #{path_to_pdk} new class foo") do
```

Comment:
@jessereynolds work here might help with this:
https://github.com/puppetlabs/pdk/pull/62/files#diff-174f7dfb508155027cb4c3e9a1840591R28